docs/src/running-validator/validator-troubleshoot.md

```diff
@@ -15,3 +15,31 @@ testnet participants, [https://discord.gg/pquxPsq](https://discord.gg/pquxPsq).
 - [Core software repo](https://github.com/solana-labs/solana)
 
 Can't find what you're looking for? Send an email to ryan@solana.com or reach out to @rshea\#2622 on Discord.
+
+## Blockstore
+
+The validator blockstore rocksdb database can be inspected using the `ldb` tool.
+`ldb` is part of the `rocksdb` code base and is also available in the `rocksdb-tools`
+package.
+
+[RocksDB Administration and Data Access Tool](https://github.com/facebook/rocksdb/wiki/Administration-and-Data-Access-Tool)
+
+## Downgrade
+
+If a new column family has been introduced to the validator blockstore a
+subsequent downgrade of the validator to a version prior to the version where
+the new column family was introduced will cause the validator to fail while
```

Comment:
nit: "to a version prior to the version where the new column family was introduced will" ==> "to a version that predates the new column family will ..."

My attempt to make it slightly less wordy, feel free to ignore if you like what you already have more.